src/main/java/frc/robot/Constants.java

```diff
@@ -468,7 +468,7 @@ public static final class CraneConstants {
       kElevatorMaxSpeedMetersPerSecond / kAccelerationSeconds; // Do not change.
 
     public static final SparkUtil.PIDFSlot kPivotMotorVelocityPIDFSlot = new SparkUtil.PIDFSlot(
-      new PIDF(0.01, 0.0, 0.0, 0.0), // TODO: Tune.
+      new PIDF(0.1, 0.0, 0.0, 0.0), // TODO: Tune.
```

Comment:
We use `TODO` to denote code that needs additional work. When we complete that work, `TODO` should be removed.